pyproject.toml

```diff
@@ -0,0 +1,12 @@
+[project]
+name = "fakeiteasy"
+version = "0.1.0" # value isn't important; we're just building the documentation
+requires-python = ">=3.13"
+
+[tool.uv]
+dev-dependencies = [
+    "linkchecker>=10.5.0",
+    "mike>=2.1.3",
+    "mkdocs>=1.6.1",
+    "mkdocs-material>=9.5.39",
+]
```

Comment:
Oops. I accidentally upgraded everything while I was at it!

Reply:
toml 🤢 
I'm not sure if I hate it more or less than yaml ^^